### ISSUE TITLE
FAQ extension breaking locale behaviour

### ISSUE THREAD

**Issue by eliotsykes** [joshnuss/spree-faq#2]
## Josh is aware of this, and acknowledged he'll take a look at merging, I'm just reporting this here so in the meantime other developers are saved time. Copied from http://groups.google.com/group/spree-user/browse_thread/thread/c7587eb8f0440425?hl=en

I've just been adding the FAQ extension to my store (which is a great 
extension btw) and found a bug which caused locale problems. 
Here's the change that has fixed the bug: 

http://github.com/eliotsykes/spree_faq/commit/7c6bbf8999ce0d70880666b3b7bca99b0afa336c

In short, there was some code to modify a Controller outside of the 
extension's activate method.  The fix was to put the code inside the 
activate method.

**Comment by joshnuss** [comment]
Fixed
